imap_processing/cli.py

```diff
@@ -101,6 +101,29 @@ def _parse_args():
     )
 
     parser = argparse.ArgumentParser(prog="imap_cli", description=description)
+    # TODO: Add version here and change our current "version" to "data-version"?
```

Comment:
I'm not sure how hard this would be to change, but most CLIs allow `cli --version` to get the cli version resolution. That would be nice here if possible, but figured I wouldn't change this right this moment.

Reply:
I like that

Reply:
To your question about how hard it is to make this, we will need to change here and update batch starter lambda and its test to use `data-version` instead of `version`. I think it will be ok.

Reply:
OK, do you want me to do that now, or hold off until after SIT-3 then?

Reply:
I thought we already were using `data-version`? If we are using `version` then yeah leave it for later, removing/changing functionality from the API is annoying